auth-web/src/views/auth/home/IncorpOrRegisterView.vue

```diff
@@ -38,31 +38,65 @@
-                              {{subItem.text}}
-                            </v-list-item-subtitle>
-                          </v-list-item-content>
-                        </v-list-item>
-                      </v-list-item-content>
+                  <v-expansion-panel-content class="list-item-text py-3">
+                    <span id="expPnlContent1">The name(s) and address(es) of the proprietor or partner(s).</span>
+                  </v-expansion-panel-content>
+                </v-expansion-panel>
+                <v-tooltip top max-width="450px" light content-class="tooltip">
+                  <template v-slot:activator="{ on }">
+                    <v-expansion-panel class="incorp-expansion-panels">
+                      <v-expansion-panel-header class="incorp-expansion-header font-weight-bold">
+                        <template v-slot:actions>
+                          <v-icon color="primary">
+                            $expand
+                          </v-icon>
+                        </template>
+                        <span class="tooltip-text" v-on="on">B.C. Based Company</span>
+                      </v-expansion-panel-header>
+                      <v-expansion-panel-content class="list-item-text py-3">
+                        <span id="expPnlContent2">Office addresses, director names and addresses, share structure and
+                          articles.</span>
+                      </v-expansion-panel-content>
+                    </v-expansion-panel>
+                  </template>
+                  <v-card class="tooltip-content">
+                    <div>
+                      <h3 class="mb-3">B.C. Based Company</h3>
+                      <span>You can incorporate the following B.C. based company types: Limited Company, Unlimited
+                        Liability Company, Benefit Company, and Community Contribution Company.</span>
+                    </div>
+                  </v-card>
+                </v-tooltip>
+                <v-expansion-panel class="incorp-expansion-panels">
+                  <v-expansion-panel-header class="incorp-expansion-header font-weight-bold">
+                    <template v-slot:actions>
+                      <v-icon color="primary">
+                        $expand
+                      </v-icon>
+                    </template>
+                    Cooperative Association
+                  </v-expansion-panel-header>
+                  <v-expansion-panel-content class="list-item-text py-3">
+                    <span id="expPnlContent3">Office addresses, director names and addresses, rules of the association
+                      and memorandum.</span>
                   </v-expansion-panel-content>
                 </v-expansion-panel>
               </v-expansion-panels>
             </div>
+
+            <!-- For BEN only as feature flag 'EnableBcCccUlc' enabled. Remove section when FF removed -->
```

Comment:
Did you create a tech debt ticket to remove the obsolete code and feature flag when this feature is released to Prod?

If so then I think you don't need the extra comment here.

Reply:
I don't think I have yet but will do right away

Reply:
Before:
![image](https://user-images.githubusercontent.com/116035339/207987322-57a260a3-0bd8-4691-baf1-41f3d19c2c46.png)


After:
![image](https://user-images.githubusercontent.com/116035339/207987234-cd99e9d8-61b9-484e-89e3-2781af997e30.png)